### PR TITLE
Add additional footer information and fix box shadow and remove last line from dataset list

### PR DESCRIPTION
**Pull request by maniarathi**
Screen Shot 2020-09-09 at 12 21 42 PM

### CODE REVIEW

File: frontend/src/components/Footer/index.tsx [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+import { MainWrapper, StyledAnchor, Wrapper } from "./style";
 
 const Footer: FC = () => {
   return (
     <Wrapper>
-      <MainWrapper>© 2020 cellxgene</MainWrapper>
+      <MainWrapper>
+        © 2020 cellxgene
+        <StyledAnchor
+          key={"mailto:cellxgene@chanzuckerberg.com"}
```
Comment (tihuan) [inline frontend/src/components/Footer/index.tsx:10]: nit: oh we only need `key` when we're generating components in a `map()` call 😄

File: frontend/src/components/Footer/index.tsx [hunk 2]
```diff
@@ -1,10 +1,44 @@
 import React, { FC } from "react";
-import { MainWrapper, Wrapper } from "./style";
+import { MainWrapper, StyledAnchor, Wrapper } from "./style";
 
 const Footer: FC = () => {
   return (
     <Wrapper>
-      <MainWrapper>© 2020 cellxgene</MainWrapper>
+      <MainWrapper>
+        © 2020 cellxgene
```
Comment (tihuan) [inline frontend/src/components/Footer/index.tsx:8]: nit: For future proof, we can do: 

```ts
<MainWrapper>© {new Date().getFullYear()} cellxgene</MainWrapper>
```

**Review by tihuan — APPROVED** [review]
Looks super good! Thanks for updating this quickly, @maniarathi 🙏 ✨ !! Small nits otherwise LGTM!

File: frontend/src/components/Footer/index.tsx (hunk 1 above)
Comment (maniarathi) [reply frontend/src/components/Footer/index.tsx:10]: Ahh oops! Deleted!

File: frontend/src/components/Footer/index.tsx (hunk 2 above)
Comment (maniarathi) [reply frontend/src/components/Footer/index.tsx:8]: Ooh fancy! Thanks!

**Pull request merged**